app/controllers/api/v1/books_controller.rb

```diff
@@ -1,30 +1,25 @@
 class Api::V1::BooksController < ApplicationController
+  before_action :authenticate_user!
   before_action :set_book, only: %i[ show update destroy ]
-
-  # GET /books
+  
   def index
-    @books = Book.all
-
+    @books = current_user.books
     render json: @books
   end
 
-  # GET /books/1
   def show
     render json: @book
   end
-
-  # POST /books
+  
   def create
-    @book = Book.new(book_params)
-
+    @book = current_user.books.build(book_params)
```

Comment:
@nshelokar-vf unnecessary declaration of instance variable.

Reply:
Removed it.